app/vmctl/vm_native.go

```diff
@@ -239,7 +248,7 @@ func (p *vmNativeProcessor) runBackfilling(ctx context.Context, tenantID string,
 			case infErr := <-errCh:
 				return fmt.Errorf("native error: %s", infErr)
 			case filterCh <- native.Filter{
-				Match:     fmt.Sprintf("{%s=%q}", nameLabel, s),
+				Match:     s,
```

Comment:
Probably, it would be better from code readability and maintainability PoV constructing the final `Match` filter value here instead of overriding it later at worker side?

Reply:
fixed